src/util/Permissions.js

```diff
@@ -57,7 +57,8 @@ class Permissions extends BitField {
  * * `ADD_REACTIONS` (add new reactions to messages)
  * * `VIEW_AUDIT_LOG`
  * * `PRIORITY_SPEAKER`
- * * `STREAM`
+ * * `VIDEO` (allows go live and video calls)
```

Comment:
This should really just say:
```suggestion
 * * `VIDEO`
```

Because the one below thats deprecated already mentions to use video instead, and if you look for the video permission, you are on the right track already anyway.